app/models/concerns/smooch_search.rb

```diff
@@ -304,7 +306,8 @@ def ask_for_feedback_when_all_search_results_are_received(app_id, language, work
       end
     end
 
-    def search_for_explainers(uid, query, team_id, language = nil)
+    def search_for_explainers(uid, query, team_id, language = nil, limit = nil)
+      limit ||= CheckConfig.get(:most_relevant_team_limit, 3, :integer)
```

Comment:
As I said in another comment, I suggest we always enforce a `limit` to be passed, instead of accepting `nil` and falling back to a default value.

Reply:
Done